packages/bpk-tokens/src/android/base/aliases.json

```diff
@@ -20,8 +20,6 @@
     "LINE_HEIGHT_XXL": 40,
     "FONT_WEIGHT_REGULAR": "400",
     "FONT_WEIGHT_MEDIUM": "500",
-    "FONT_WEIGHT_BOLD": "700",
-    "FONT_WEIGHT_BLACK": "900",
```

Comment:
These were unused - aliases are not included in output (apart from raw but these token formats are not "advertised") so non breaking change.